### ISSUE TITLE
delete() function for 'blood_pressure' type causes crash

### ISSUE THREAD

**Issue by rafaellop** [dariosalvi78/cordova-plugin-health#266]
When trying to delete an entry in the Apple Health using the following:

`navigator.health.delete({...})`

and when the key is 'blood_pressure' then this error is logged:

> *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'Authorization to read the following types is disallowed: HKCorrelationTypeIdentifierBloodPressure' terminating with uncaught exception of type NSException

To avoid the issue delete 'blood_pressure_systolic' and 'blood_pressure_diastolic' separately.

**Comment by dariosalvi78** [comment]
I suppose you got R/W access for blood pressure?

**Comment by rafaellop** [comment]
Yes, of course. I suppose it's because a similar issue like your fix addressed:
https://github.com/dariosalvi78/cordova-plugin-health/commit/f27d2cb69cf4cfda6bddd4b2167dc58ac4e8d47d

**Comment by dariosalvi78** [comment]
not sure it's related. Have you tried deleting it with the telerik's plugin? (it's available if you install this plugin too)

**Comment by rafaellop** [comment]
No, I haven't tried telerik plugin. I just removed both values (systolic and disatolic) separately and this works.

**Comment by dariosalvi78** [comment]
maybe correlation type values cannot be deleted? Can you try storing a `nutrition` value and see if it can be deleted?
